inc/classes/class_func.php

```diff
@@ -200,67 +195,56 @@ public function unixstamp2date($func_timestamp, $func_art)
                         $day[5] = t('Fr');
                         $day[6] = t('Sa');
 
-                        $func_date .= $day[date("w", $func_timestamp)];
-                        $func_date .= ", ";
-                        $func_date .= date("H:i", $func_timestamp);
+                        $func_date = $day[date('w', $func_timestamp)];
+                        $func_date .= ', ';
+                        $func_date .= date('H:i', $func_timestamp);
                     break;
             }
         }
-          return $func_date;
-    }
 
-  /**
-   * Calculates the age to a given birthday timestamp
-   * @timestamp birthhday
-   * @optional timestamp attime Calculates the age the person will have at this time
-   * @return int age
-   */
-    public function age($birthday, $attime = 0)
```

Comment:
I'm wondering:
* Why a user-age related function is in `func` and not in `user`
* Why this is not used anywhere, although there has to be functionality to evaluate the minimum age. Duplicated code?
* Why [DateTime::diff](http://php.net/manual/de/datetime.diff.php) isn't used to provide this. Guess this is pre-PHP-5.3 developed code.

I agree with the removal from `func`, but would suggest to have a look at the current age verification system, if a similar function in user exists (or should be created)

Reply:
There is already an option to define a minimum age for a party and AFAIK this is also enforced on signup, so similar code must already exist at another place. But I would also need to search for it first.

But as said not blocking the removal here, may just be worth a look.